### ISSUE TITLE
error: unknown type name ‘GElf_Versym’; did you mean ‘GElf_Verdef’?

### ISSUE THREAD

**Issue by NobinPegasus** [iovisor/bcc#4911]
While trying to build the BCC I face the following error:
```
/home/pegasus/Documents/eBPF/learning-ebpf/bcc/src/cc/libbpf/src/elf.c:160:9: error: unknown type name ‘GElf_Versym’; did you mean ‘GElf_Verdef’?
  160 |         GElf_Versym versym;
      |         ^~~~~~~~~~~
      |         GElf_Verdef
```
How can I fix it?

**Comment by yanruotian** [comment]
Hello I meet the same error. May I have your suggestion if you have fixed it? Thank you.

**Comment by NobinPegasus** [comment]
Hello, you can look here. It was a mismatch in versions of libelf-dev, llvm, clang. If I recall it right. So what you have to do is do a complete fresh installation of all LLVM related packages. And be sure that the versions all match. I would suggest building from source. And make sure to add the build to the path.

**Comment by yanruotian** [comment]
Yes you are right. I reinstalled all related packages and the error disappeared. Thank you very much.
